app/controllers/rooms_controller.rb

```diff
@@ -0,0 +1,2 @@
+class RoomsController < ApplicationController
+end
```

Comment:
Since we are working with API only application, you could delete this controller to follow the one that has the versioning: Api/v1

Reply:
👍